packages/babel-preset-gatsby/src/index.ts

```diff
@@ -29,7 +29,12 @@ export function loadCachedConfig() {
   return pluginBabelConfig
 }
 
-export default function preset(_, options = {}) {
+interface IOptions {
+  targets?: Browserslist | { node: `current` } | string
+  stage?: string
```

Comment:
It's failing typechecks because this also needs `reactRuntime?: "classic" | "automatic"`.